README.md

```diff
@@ -4,11 +4,32 @@ Library for validating Canadian Postal Codes and US Zip Codes
 
 # usage
 
-yarn add zip-postal-codes
+`yarn add zip-postal-codes`
 
+```javascript
+1. CA
+> import { isValidateCaPostalCodes } from 'zip-postal-codes/src';
+<strong> Province </strong>: should be the first 2 letters or the the full name of the province
+<strong> Postal Code </strong>: only the first letter 
+    isValidateCaPostalCodes(province: string, postalCode: string); 
```

Comment:
We should probably change the name of `isValidate` for the followings

1. `isValidate` isn't grammatically correct
2. `isValidateCaPostalCodes` is implying plural but we only pass one province.
3. `isValidateCaPostalCodes` is a long function name. Not great for JavaScript.

Maybe something like

`validatePostal(province, postalCode)`